tests/BaseElementTest.php

```diff
@@ -484,7 +484,7 @@ public function testPreviewLink(string $class, string $elementIdentifier, ?strin
         $previewLink = $element->PreviewLink();
```

Comment:
I'd do 
`$previewLink = $element->PreviewLink() ?? '';`
and then we don't need to worry about it for the whole method.

Reply:
Updated